backend/apps/core/api/algolia.py

```
@@ -52,11 +53,17 @@ def algolia_search(request):
 
         facet_filters = data.get("facetFilters", [])
         index_name = data.get("indexName")
-        ip_address = get_user_ip_address(request)
-        limit = int(data.get("hitsPerPage", 25))
-        page = int(data.get("page", 1))
+
+        limit = data.get("hitsPerPage", 25)
+        page = data.get("page", 1)
+        ip_address = get_user_ip_address(request)       
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)